.github/workflows/ci.yml

```diff
@@ -0,0 +1,17 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    - cron: '20 14 * * 2'
```

Comment:
Why are we doing this on a cron?
How was the period chosen?

Reply:
> Why are we doing this on a cron?
Matches what happens on travis, which also runs on a weekly cron (daily for installer + kitchen-sink). That's the main way we can tell if builds are broken

> How was the period chosen?
https://github.com/emteknetnz/module-standardiser/blob/main/WorkflowCreator.php#L86